tests/test_niriss_imaging_modes.py

```diff
@@ -0,0 +1,47 @@
+"""System test of mirage/NIRISS for regular and NRM imaging.
+
+Authors
+-------
+    - Kevin Volk
+
+Use
+---
+    >>> pytest -s test_niriss_imaging_modes.py
```

Comment:
You should be able to use this command from the command line in order to run this test using pytest.